pkg/agent/route/route_windows.go

```diff
@@ -364,8 +442,14 @@ func (c *Client) listRoutes() (map[string]*util.Route, error) {
 		if !rt.DestinationSubnet.IP.IsGlobalUnicast() {
 			continue
 		}
-		// Windows adds an active route entry for the local broadcast address automatically when a new IP address
-		// is configured on the interface. This route entry should be ignored in the list.
+		// When configuring an IP address to an interface on Windows, three route entries will be automatically added.
+		// For example, if the IP address is 10.10.0.1/24, the following three routes will be created:
+		// Network Destination   Netmask          Gateway  Interface  Metric
+		// 10.10.0.1             255.255.255.255  On-link  10.10.0.1  281
+		// 10.10.0.0             255.255.255.0    On-link  10.10.0.1  281
+		// 10.10.0.255           255.255.255.255  On-link  10.10.0.1  281
+		// The host (10.10.0.1) and broadcast (10.10.0.255) routes should be ignored since they are unnecessary and redundant.
```

Comment:
```suggestion
		// The host (10.10.0.1) and broadcast (10.10.0.255) routes should be ignored since they are not supposed to be managed by Antrea.
```

Reply:
Done